Spawn-App-iOS-SwiftUI/ViewModels/ChoosingTags/ChoosingTagViewModel.swift

```diff
@@ -40,5 +41,34 @@ class ChooseTagPopUpViewModel: ObservableObject {
 		}
 	}
     
-    //TODO: add API call to fetch tags
+//    func fetchTagsToAddToFriend(friendUserId: UUID, friendTagIds: [UUID]) async {
+//        let urlString = APIService.baseURL + "friendTags/addUserToTags"
```

Comment:
```suggestion
//        let urlString = APIService.baseURL + "friendTags/addUserToTags/\(userId)"
```
As suggested above, let's take in the user id to the constructor and use it here